Assets/MRTK/Providers/LeapMotion/Editor/LeapMotionDeviceManagerProfileInspector.cs

```diff
@@ -0,0 +1,93 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See LICENSE in the project root for license information.﻿
+
+using Microsoft.MixedReality.Toolkit.Editor;
+using Microsoft.MixedReality.Toolkit.LeapMotion.Input;
+using Microsoft.MixedReality.Toolkit.LeapMotion.Utilities;
+using Microsoft.MixedReality.Toolkit.Utilities.Editor;
+using System.Linq;
+using UnityEditor;
+
+namespace Microsoft.MixedReality.Toolkit.LeapMotion.Inspectors
+{
+    [CustomEditor(typeof(LeapMotionDeviceManagerProfile))]
+    /// <summary>
+    /// Custom inspector for the Leap Motion input data provider
+    /// </summary>
+    public class LeapMotionDeviceManagerProfileInspector : BaseMixedRealityToolkitConfigurationProfileInspector
+    {
+        protected const string ProfileTitle = "Leap Motion Controller Settings";
+        protected const string ProfileDescription = "";
+
+        protected LeapMotionDeviceManagerProfile instance;
+        protected SerializedProperty leapControllerOrientation;
+        protected SerializedProperty leapControllerOffset;
+
+        protected override void OnEnable()
+        {
+            base.OnEnable();
+
+            instance = (LeapMotionDeviceManagerProfile)target;
+            leapControllerOrientation = serializedObject.FindProperty("leapControllerOrientation");
+            leapControllerOffset = serializedObject.FindProperty("leapControllerOffset");
+        }
+
+        /// <summary>
+        /// Display the MRTK header for the profile and render custom properties
+        /// </summary>
+        public override void OnInspectorGUI()
+        {
+            RenderProfileHeader(ProfileTitle, ProfileDescription, target);
+
+            // Add the documentation help button
+            using (new EditorGUILayout.HorizontalScope())
+            {
+                // Draw an empty title to align the documentation button to the right
+                InspectorUIUtility.DrawLabel("", InspectorUIUtility.DefaultFontSize, InspectorUIUtility.ColorTint10);
+
+                var helpURL = "https://microsoft.github.io/MixedRealityToolkit-Unity/Documentation/CrossPlatform/LeapMotionMRTK.html";
+
+                InspectorUIUtility.RenderDocumentationButton(helpURL);      
+            }
+
+            RenderCustomInspector();
+        }
+
+        /// <summary>
+        /// Render the custom properties for the Leap Motion profile
+        /// </summary>
+        public virtual void RenderCustomInspector()
+        {
+            using (new EditorGUI.DisabledGroupScope(IsProfileLock((BaseMixedRealityProfile)target)))
+            {
+                // Show warning if the leap core assets are not in the project
+                if (!LeapMotionUtilities.IsLeapInProject)
+                {
+                    EditorGUILayout.HelpBox("The Leap Motion Core Assets could not be found in your project. For more information, visit the Leap Motion MRTK documentation.", MessageType.Error);
```

Comment:
Would it be possible to add a documentation button here? Something like https://github.com/microsoft/MixedRealityToolkit-Unity/blob/40988308135e59bcc79b30823e56348851963437/Assets/MRTK/Providers/WindowsMixedReality/Shared/Editor/WindowsMixedRealityCameraSettingsProfileInspector.cs#L47

Reply:
Yep! Moved

Reply:
Oh! Completely missed that you had a button in a different spot. Thanks for moving!